src/workspace-hack/build.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
why did you add a buildscript?

Reply:
Added a comment! It's required for Cargo to honor build dependencies.